templates/js/_github/workflows/release.yml

```diff
@@ -0,0 +1,17 @@
+name: Deploy '%APPNAME%' App Components to Mainnet
```

Comment:
Also pull from quickstart -- we can replace deploy, signer with app name

https://github.com/NEARBuilders/quickstart/blob/main/.github/workflows/release-mainnet.yml

Maybe some future work is creating the access keys on init @bos-cli-rs